ui/src/app/shared/models.ts

```diff
@@ -657,7 +668,7 @@ export interface HelmChart {
     versions: string[];
 }
 
-export type AppSourceType = 'Helm' | 'Kustomize' | 'Directory' | 'Plugin';
+export type AppSourceType = 'Helm' | 'Kustomize' | 'Directory' | 'Plugin' | 'OCI';
```

Comment:
Is this necessary? As far as I can tell, OCI isn't a source "type" in the same way that Helm or Kustomize are. An OCI repo could contain Helm, Kustomize, Directory, or Plugin-style manifetss.

Reply:
Yeah there's probably no need for that actually - let me doublecheck.

Reply:
Indeed, this is unnecessary